openedx/core/djangoapps/external_user_ids/admin.py

```diff
@@ -0,0 +1,10 @@
+from django.contrib import admin
```

Comment:
Noob question: Does having this make it so you can run the management command from Django admin? Is this required for the overall management command? or is it a nice to have? 

As in, without this piece, can you still run the command solely from a command line via manage.py?

Reply:
Let's chat about this one more in person, but the main goal of the admin page here is to be able to set the configuration and list of users we need to generate external IDs for.

Reply:
Thanks for the explanation! 👍